src/index.css

```diff
@@ -9,21 +9,256 @@ ul {
 }
 
 body {
-  background-color: #050518;
+  background-color: #fafafa;
   font-family: sans-serif;
   overflow: auto;
-  color: #fff;
+  color: #000;
 }
 
 .navbar {
   display: flex;
+  align-items: center;
   justify-content: space-between;
+  height: 5.938rem;
+  width: 100vw;
+  padding: 3rem 6rem;
+  border-bottom: 1px solid rgb(211, 205, 205);
+  background: rgb(255, 255, 255);
+}
+
+.Line {
+  width: 84rem;
+  height: 0.125rem;
+  margin: 0.93rem 6rem;
+  border: solid 1px #e8e8e8;
+}
+
+.footer {
+  margin: 0 6rem;
+}
+
+.add-book {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.logo-links {
+  display: flex;
+  gap: 25px;
   align-items: center;
-  padding: 1rem 2rem;
-  background-color: #050518;
-  border-bottom: 1px solid #616164;
-  box-shadow: #b8b2b2 0 0 10px 0;
+}
+
+.nav-categories {
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212 !important;
+}
+
+.nav-book {
+  height: 1rem;
+  letter-spacing: 1.9px;
+  color: #121212 !important;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+form input,
+.choice {
+  width: 25rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  color: #121212;
+  padding: 0.5rem;
+}
+
+.book {
+  width: 92rem;
+  display: flex;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 1rem;
+  margin: 0.93rem 1rem;
+  height: 10.6rem;
+  background-color: #fff;
+  border-radius: 5px;
+  align-items: center;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  color: rgb(47, 136, 238);
+  font-size: 2.5rem;
+  font-weight: bolder;
+}
```

Comment:
- [ ] Well done implementing the required changes, but it should look EXACTLY like the Zeplin design 👇🏽 
- [ ] **[OPTIONAL]** Please use the correct font family designed in the original design it is Montserrat.

| Zeplin design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/232245072-48dadf8a-4e46-40fd-ba24-6626cc2b5ca5.png) | ![image](https://user-images.githubusercontent.com/58642949/232245095-14bd918f-24d1-497e-8040-b755de5c352d.png) |

Reply:
The problem is occurring when viewing the UI from a smaller devices. I will resolve the issue 👍